usr/src/cmd/cmd-inet/usr.sbin/Makefile

```diff
@@ -66,12 +66,12 @@ SRCS=		$(PROGSRCS) $(OTHERSRC)
 
 SUBDIRS=	in.rdisc \
 		in.talkd inetadm inetconv ipmpstat ipqosconf \
-		nwamadm nwamcfg routeadm \
+		nwamadm nwamcfg routeadm snoop \
```

Comment:
I'd like to ask @jclulow and @citrus-it to take a look at this, because snoop has been such a hot bed of security problems where it reads untrusted data as root so much.

Reply:
I redid the snoop patch a bit its now cleaner then the previous one mostly struct timeval to struct timeval32 and some
fallout from that as localtime() needs a time_t so we need an extra local variable to store the tv_sec in from the
struct timeval32 tv_sec member. Rest is the same the ntp id cast is the same a most other casts for ntp stuff seems
that one was just forgotten or was not a problem before but is now. Then there is the missing header which is also
fixed.